src/main/kotlin/note/application/NoteUpdater.kt

```diff
@@ -12,6 +12,14 @@ class NoteUpdater(
     private val repository: NoteRepository,
     private val identifierGenerator: NoteIdentifierGenerator,
 ) {
+
+    /***
+     * Updates a Note with the given identifier, with the given title and description,
+     * @see Note
+     * @throws NonExistentNoteException if the note does not exist
+     * @throws UnchangedNoteException if the note is unchanged
+     * @return null or a note instance if it exists
```

Comment:
never returns null. It allways return an instance of note if the exceptions are not thrown.

Rebase and push and we're good to go